src/Analyzers/Analyzers/src/StartupAnalyzer.Diagnostics.cs

```diff
@@ -38,7 +38,7 @@ static Diagnostics()
 
             internal readonly static DiagnosticDescriptor UnsupportedUseMvcWithEndpointRouting = new DiagnosticDescriptor(
                 "MVC1005",
-                "Cannot use UseMvc with Endpoint Routing.",
+                "Cannot use UseMvc with Endpoint Routing",
```

Comment:
I _think_ it may be considered a breaking change to update existing diagnostic messages. Why are all the final periods going away❔

Reply:
I believe some new analyzer got picked up as part of the M.C.A update that I did, although I'm not sure which one of the dependencies is pulling in `Microsoft.CodeAnalysis.Analyzers`.

Reply:
All gone now 😀